Userland/Shell/Formatter.cpp

```diff
@@ -15,7 +16,8 @@ namespace Shell {
 
 DeprecatedString Formatter::format()
 {
-    auto node = m_root_node ? m_root_node : Parser(m_source).parse();
+    auto node = m_root_node ?: m_parse_as_posix ? Posix::Parser(m_source).parse()
```

Comment:
not immediately obvious when reading which of two options is parsed here:
```
    auto node = (m_root_node ?: m_parse_as_posix) ? Posix::Parser(m_source).parse()
```
or
```
    auto node = m_root_node ?: (m_parse_as_posix ? Posix::Parser(m_source).parse())
```

(probably the latter but clarifying brackets could help)

Reply:
:shrug: looks clear to me, but sure.